scripts/build-images

```diff
@@ -71,5 +71,14 @@ xargs -n1 -t docker image pull --quiet << EOF > build/images-multus.txt
     ${REGISTRY}/rancher/hardened-sriov-network-webhook:v1.0.0-build20210429
 EOF
 
+xargs -n1 -t docker image pull --quiet << EOF > build/images-harvester.txt
+    ${REGISTRY}/rancher/harvester-cloud-provider:v0.1.1
+    ${REGISTRY}/rancher/harvester-csi-driver:v0.1.1
+    ${REGISTRY}/longhornio/csi-node-driver-registrar:v1.2.0-lh1
+    ${REGISTRY}/longhornio/csi-resizer:v0.5.1-lh1
+    ${REGISTRY}/longhornio/csi-provisioner:v1.6.0-lh1
+    ${REGISTRY}/longhornio/csi-attacher:v2.2.1-lh1
```

Comment:
All airgapped images need to be in the rancher repo. You might need to get these copied in via a PR to the rancher/image-mirror repo.